kunai-lib/lib/DEX/analysis/basic_blocks.cpp

```diff
@@ -13,7 +13,8 @@ using namespace KUNAI::DEX;
 void BasicBlocks::remove_node(DVMBasicBlock *node)
 {
     // with this we provide RAII
-    std::unique_ptr<DVMBasicBlock> node_ (node);
+    // TODO: Remove unused variable
+    std::unique_ptr<DVMBasicBlock> const node_ (node);
```

Comment:
The node_ variable should be removed in the future, as it is not used.
I understand the purpose of the RAII.